### PR TITLE
minor change: correct indentation

### LINKED ISSUE #8: .pdsc: correct indentation

### PR DESCRIPTION
**Pull request by gorazdko**
Resolve #8

### CODE REVIEW

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc [hunk 1]
```diff
@@ -18,13 +18,13 @@
       <book category="other" name="docs/L-Tek-A4-Technical-Specification-FF-LPC546XX-15.pdf" title="Manual" />
       <book category="other" name="docs/L-Tek-A4-Inlay-FF-LPC546XX-10.pdf" title="Pinout" /> 
       <mountedDevice    Dname="LPC54606J512" Dvendor="NXP:11"/>
-      <compatibleDevice Dname="LPC54608J512" Dvendor="NXP:11" />	  
+      <compatibleDevice Dname="LPC54608J512" Dvendor="NXP:11" />
```
Comment (nejchi31) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: Wrong indentation.

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc [hunk 2]
```diff
@@ -16,9 +16,9 @@
       <description>The L-Tek FF-LPC546XX is a small, 40-pin development board intended for prototyping general microcontroller applications. It is based on a low power, peripheral rich and debug capable NXP LPC54606 microcontroller with integrated DAPLink interface for complete debugging experience.</description>
       <image small="img/ff_lpc546xx.png"  large="img/ff_lpc546xx.png"/>
       <book category="other" name="docs/L-Tek-A4-Technical-Specification-FF-LPC546XX-15.pdf" title="Manual" />
```
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termiantion

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc [hunk 3]
```diff
@@ -16,9 +16,9 @@
...
-      <book category="other" name="docs/L-Tek-A4-Inlay-FF-LPC546XX-10.pdf" title="Pinout" /> 
+      <book category="other" name="docs/L-Tek-A4-Inlay-FF-LPC546XX-10.pdf" title="Pinout" />
```
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termiantion

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc (hunk 2 above)
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termiantion

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc (hunk 3 above)
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termiantion

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc [hunk 4]
```diff
@@ -16,9 +16,9 @@
...
       <mountedDevice    Dname="LPC54606J512" Dvendor="NXP:11"/>
-      <compatibleDevice Dname="LPC54608J512" Dvendor="NXP:11" />      
+      <compatibleDevice Dname="LPC54608J512" Dvendor="NXP:11" />
```
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termination

File: CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc (hunk 2 above)
Comment (klemen-skoda) [inline CMISIS-PACK/REPO/L-Tek.FF_LPC546XX_BSP.pdsc]: remove space before element termination, lines: 18,19,21,26

**Pull request merged**